config/src/main/java/com/alibaba/nacos/config/server/service/dump/DumpService.java

```diff
@@ -194,29 +190,23 @@ void handleConfigDataChange(Event event) {
      */
     protected abstract void init() throws Throwable;
     
-    void clearConfigHistory() {
-        LOGGER.warn("clearConfigHistory start");
-        if (canExecute()) {
-            try {
-                Timestamp startTime = getBeforeStamp(TimeUtils.getCurrentTime(), 24 * getRetentionDays());
-                int pageSize = 1000;
-                LOGGER.warn("clearConfigHistory, getBeforeStamp:{}, pageSize:{}", startTime, pageSize);
-                historyConfigInfoPersistService.removeConfigHistory(startTime, pageSize);
-            } catch (Throwable e) {
-                LOGGER.error("clearConfigHistory error : {}", e.toString());
-            }
-        }
-        
-    }
-    
     /**
      * config history clear.
      */
     class ConfigHistoryClear implements Runnable {
         
+        private HistoryConfigCleaner historyConfigCleaner;
+        
+        public ConfigHistoryClear(HistoryConfigCleaner historyConfigCleaner) {
+            this.historyConfigCleaner = historyConfigCleaner;
+        }
+        
         @Override
         public void run() {
-            clearConfigHistory();
+            LOGGER.warn("clearHistoryConfig start");
```

Comment:
1.when canExecute return false/true ,print log also ,like 'history config cleaner is not enable in current context' , 'history config cleaner is enable in current context, trying to run cleaner.'
2.when cleaner run finish ,print log like 'history cleaner finished successfully' and should catch exception when  cleaner throw exception ,or it wont be scheduled next period.